src/Microsoft.Azure.CosmosRepository/Repositories/DefaultRepository.Paging.cs

```diff
@@ -3,6 +3,10 @@
 
 
 // ReSharper disable once CheckNamespace
+using System.Threading;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Extensions.Options;

```

Comment:
These should be moved to the [_GlobalUsings.cs_ file](https://github.com/IEvangelist/azure-cosmos-dotnet-repository/blob/main/src/Microsoft.Azure.CosmosRepository/GlobalUsings.cs).